src/CoreBundle/DependencyInjection/ChameleonSystemCoreExtension.php

```diff
@@ -174,7 +174,23 @@ private function addMailerConfig(array $mailerConfig, ContainerBuilder $containe
             return;
         }
         $mailerDefinition = $container->getDefinition('chameleon_system_core.mailer');
-        $mailerDefinition->addMethodCall('setSmtpHost', array($mailerConfig['host']));
+
+        $host = $mailerConfig['host'];
+        $port = (int) $mailerConfig['port'];
+        if (false !== ($portPosition = \mb_strrpos($host, ':'))) {
+            $portInHost = \mb_substr($host, $portPosition + 1);
+            if (true === \is_numeric($portPosition) && (25 === $port || $port === (int) $portInHost)) {
```

Comment:
The first comparison is superfluous, isn't it? We checked in line 180 if it is false, so it must be numeric here. Maybe you wanted to check `\is_numeric($portInHost)`?

Reply:
true - that was a silly mistake